src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -221,12 +221,16 @@ private Schema buildLoadDatasetSchema() {
         return Schema.of(fieldList);
     }
 
-    private static final String insertLoadHistoryToStagingTableTemplate =
+    public static final String insertLoadHistoryToStagingTableTemplate =
         "INSERT INTO `<project>.<dataset>.<stagingTable>`" +
             " (load_tag, load_time, source_name, target_path, state, file_id, checksum_crc32c, checksum_md5, error)" +
             " VALUES <load_history_array:{v|('<load_tag>', '<load_time>', '<v.sourcePath>', '<v.targetPath>'," +
             " '<v.state>', '<v.fileId>', '<v.checksumCRC>', '<v.checksumMD5>', '<v.error>')}; separator=\",\">";
 
+    public String sanitizeErrorMsg(String error) {
+        return error.replaceAll("'", "\'");
```

Comment:
That backslash needs to make it all the way into BigQuery. I _think_ it might, but I don't know what ST does to strings.

Reply:
yeah--you are right ST seems to just bypass the escape characters